package.json

```diff
@@ -22,7 +22,7 @@
     "build-index": "npm run build-package && node tasks/generate-index",
     "build-legacy": "shx rm -rf build && npm run build-index && webpack --config config/webpack-config-legacy-build.js && cleancss --source-map src/ol/ol.css -o build/legacy/ol.css",
     "copy-css": "shx cp src/ol/ol.css build/ol/ol.css",
-    "transpile": "shx rm -rf build/ol && shx mkdir -p build && buble --input src/ol --output build/ol --no modules --sourcemap",
```

Comment:
Can we remove `buble` from the devDependencies list?

Reply:
No, it's still used for building the examples.